benchmark/initialize-benchmark-entities.sh

```diff
@@ -49,11 +62,11 @@ function delete_entities {
 #main()
 #setting default values
 OPTIND=1
-REALM_NAME=realm-0
+REALM_NAME=test-realm
```

Comment:
The help above mentions that `user-0` would be the default user name. Running it without a user name on the CLI makes this script fail. IMHO it is missing to initialize the default user name here.